orchestration/dags/dependencies/firebase/sql/analytics/firebase_pro_visits.sql

```diff
@@ -0,0 +1,48 @@
+SELECT
+    session_id,
+    CONCAT(session_id, user_pseudo_id) as unique_session_id,
+    user_pseudo_id,
+    ANY_VALUE(firebase_pro_events.user_id) AS user_id,
+    user_offerer.offerer_id AS offerer_id,
+    ANY_VALUE(session_number) AS session_number,
+    MIN(event_timestamp) AS first_event_timestamp,
+    MAX(event_timestamp) AS last_event_timestamp,
+    DATE_DIFF(MAX(event_timestamp),MIN(event_timestamp),SECOND) AS visit_duration_seconds,
+
+-- page view
+    COUNTIF(event_name = "page_view" AND page_name = "Guichet - pass Culture Pro") AS nb_view_ticket_page,
+    COUNTIF(event_name = "page_view" AND page_name IN ("Offres individuelles - pass Culture Pro","Offres collectives - pass Culture Pro")) AS nb_view_offer_page,
+    COUNTIF(event_name = "page_view" AND page_name IN ("Réservations individuelles - pass Culture Pro","Réservations collectives - pass Culture Pro")) AS nb_view_booking_page,
+    COUNTIF(event_name = "page_view" AND page_name= "Remboursements - pass Culture Pro") AS nb_view_reimbursment_page,
+    COUNTIF(event_name = "page_view" AND page_name= "Statistiques - pass Culture Pro") AS nb_view_stat_page,
+
+-- offer creation
+    COUNTIF(event_name = "page_view" AND page_name="Selection du type d’offre - pass Culture Pro") AS nb_hub_for_offer_creation,
+    COUNTIF(event_name = "page_view" AND page_name= "Création - Détail de l’offre - pass Culture Pro") AS nb_start_individual_offer_creation,
+    COUNTIF(event_name = "page_view" AND page_name= "Confirmation - Offre individuelle publiée - pass Culture Pro") AS nb_confirmed_individual_offer_creation,
+    COUNTIF(event_name = "page_view" AND page_name= "Détails - Créer une offre collective vitrine - pass Culture Pro") AS nb_start_collective_offer_template_creation,
+    COUNTIF(event_name = "page_view" AND page_name= "Confirmation - Offre collective vitrine publiée - pass Culture Pro") AS nb_confirmed_collective_offer_template_creation,
+    COUNTIF(event_name = "page_view" AND page_name= "Détails - Créer une offre réservable - pass Culture Pro") AS nb_start_collective_offer_bookable_creation, 
+    COUNTIF(event_name = "page_view" AND page_name= "Confirmation - Offre réservable publiée - pass Culture Pro") AS nb_start_collective_offer_bookable_creation, 
+
+-- offer edition (next update : confirmed modification, not possible at this stage)
+    COUNTIF(event_name = "page_view" AND page_name IN ("Récapitulatif - Modifier une offre individuelle - pass Culture Pro","Edition d’une offre collective - pass Culture Pro")) AS nb_start_offer_edition,
+
+-- venue creation
+    COUNTIF(event_name IN ('hasClickedAddFirstVenueInOfferer','hasClickedAddVenueInOfferer','hasClickedCreateVenue')) AS nb_start_venue_creation,
```

Comment:
pas de event_name = "page_view" AND .... ici ? (aucune idée des arcanes du tracking mais le pattern change de tous les autres)

Reply:
@valoumiaou yesss pour ce tracker c'est exceptionnellement un Call to Action et non un page_view (les dev vont peut-être revoir ça mais je mets ça pour le moment)

Reply:
@mathilde-vey  ca marche. si ca passe en pageview  un where event_name = "page_view" simplifiera les countif